ow_plexil/src/plexil-adapter/joint_support.h

```diff
@@ -15,6 +15,11 @@
 
 const size_t NumJoints = 9;
 
+// Supports both OceanWATERS lander (6 joints) and OWLAT (7 joints).
+// Many message files define a 7-element array for both testbeds; the
+// 7th position will be ignored for OceanWATERS.
```

Comment:
This was intended from our unification document, but not the case in how it was implemented. See the wiki for details. 
https://github.com/nasa/ow_simulator/wiki/Lander-Simulation#arm-joint-telemetry

Reply:
I don't understand this comment.  Everything looks consistent to me...

Reply:
Given that the 7th position is shoulder yaw, I don't think we should say it is ignored by OceanWATERS.

Reply:
I see my confusion now.  I have removed this comment, and updated the rest of the comments in this file.